backend/core/src/main/java/org/sonarsource/sonarlint/core/BindingClueProvider.java

```diff
@@ -141,38 +170,57 @@ private Set<String> matchConnections(BindingClue bindingClue, Set<String> eligib
   }
 
   @CheckForNull
-  private static ScannerProperties extractScannerProperties(ClientFile matchedFile) {
-    LOG.debug("Extracting scanner properties from {}", matchedFile);
-    var properties = new Properties();
+  private static ConnectionProperties  extractSonarLintConfiguration(ClientFile sonarLintConfigurationFile) {
     try {
-      properties.load(new StringReader(matchedFile.getContent()));
-    } catch (Exception e) {
-      LOG.error("Unable to parse content of file '{}'", matchedFile, e);
+      var configuration = new Gson().fromJson(sonarLintConfigurationFile.getContent(), JsonObject.class);
+      var projectKey = configuration.get("projectKey");
+      var organization = configuration.get("organization");
+      var serverUrl = configuration.get("serverUrl");
```

Comment:
Is this compatible with the VS file format?

Reply:
The keys we agreed on are
`sonarCloudOrganization`, `projectKey`, `sonarQubeUri` to be compatible with VS

Reply:
Thanks, I'll fix this